libs/event/src/lib/pages/analytics/analytics.component.html

```diff
@@ -1,49 +1,41 @@
-          </mat-card>
-        </span>
-        <mat-card>
-          <article fxLayout="row" fxLayoutAlign="space-between start" flex>
-            <h3>Attendees</h3>
-            <button [disabled]="exporting" mat-button color="primary" (click)="exportTable()">
-              <ng-container *ngIf="exporting else export">
-                <span>Generating Excel...</span>
-              </ng-container>
-              <ng-template #export>
-                <mat-icon svgIcon="excel"></mat-icon>
-                <span>Export to Excel</span>
-              </ng-template>
-            </button>
-          </article>
-          <bf-table pagination="50" useFilter [source]="acceptedAnalytics">
-            <ng-template colRef="name" label="Name" let-name sort>{{ name | titlecase }}</ng-template>
-            <ng-template colRef="email" label="Email Address" let-email sort>{{ email }}</ng-template>
-            <ng-template colRef="orgName" label="Company Name" let-orgName sort>{{ orgName || dataMissing }}</ng-template>
-            <ng-template colRef="orgActivity" label="Company Activity" let-orgActivity>{{ orgActivity ? (orgActivity | toLabel: 'orgActivity') : dataMissing }}</ng-template>
-            <ng-template colRef="orgCountry" label="Country" let-orgCountry>{{ orgCountry ? (orgCountry | toLabel: 'territories') : dataMissing }}</ng-template>
-            <ng-template colRef="watchInfos.duration" label="Watch Time" let-duration sort>{{ duration * 1000 | duration }}</ng-template>
-            <ng-template colRef="watchInfos.date" label="Watching Status" let-date sort>
-              <bf-tag *ngIf="date | isWatchingNow">Watching Now</bf-tag>
+        <p>"0" means the user left the screening right away.</p>
+        <analytics-metric-card-list [cards]="aggregatedScreeningCards"></analytics-metric-card-list>
+        <article fxLayout="row" fxLayoutAlign="space-between start" flex>
+          <h3>Attendees</h3>
+          <button [disabled]="exporting" mat-button color="primary" (click)="exportTable()">
+            <ng-container *ngIf="exporting else export">
+              <span>Generating Excel...</span>
+            </ng-container>
+            <ng-template #export>
+              <mat-icon svgIcon="excel"></mat-icon>
+              <span>Export to Excel</span>
             </ng-template>
-          </bf-table>
-        </mat-card>
+          </button>
+        </article>
+        <bf-table pagination="50" useFilter [source]="acceptedAnalytics">
+          <ng-template colRef="name" label="Name" let-name sort>{{ name | titlecase }}</ng-template>
+          <ng-template colRef="email" label="Email Address" let-email sort>{{ email }}</ng-template>
+          <ng-template colRef="orgName" label="Company Name" let-orgName sort>{{ orgName || dataMissing }}</ng-template>
+          <ng-template colRef="orgActivity" label="Company Activity" let-orgActivity>{{ orgActivity ? (orgActivity | toLabel: 'orgActivity') : dataMissing }}</ng-template>
```

Comment:
```suggestion
          <ng-template colRef="orgActivity" label="Company Activity" let-orgActivity [filter]="filters.orgActivity">{{ orgActivity ? (orgActivity | toLabel: 'orgActivity') : dataMissing }}</ng-template>
```
If the displayed values are different than the one in the source data, then sometimes the filters and sorting doesn't working as expected. So we have to add a special filter function for this

You can check for example in this bf-table how it's implemented apps/festival/festival/src/app/dashboard/analytics/title/title-analytics.component.html